packages/ui-server/routes/api/init.js

```diff
@@ -31,10 +38,337 @@ router.get("/", function(req, res, next) {
 		rules: []
 	});
 
+	let usingDefaults = false;
+	let isProd = false;
+	let ExtractUseProps = [];
+	let regExpForStyles;
+
 	questioner.start({
 		action: "question",
-		question: "A dummy question for reference"
-	}).then((data) => {
+		question: Confirm("entryType", "Will your application have multiple bundles?"),
+	}).then((entryTypeAnswer) => {
+		return entryQuestions(questioner, entryTypeAnswer);
+	}).then((entryOptions) => {
```

Comment:
I guess this block is taken form the `init` package, correct?

Reply:
Yeah, whole endpoint is just JS port of init-generator @ematipico with instead of using yeoman prompts, TCP classes are used which I designed to have almost the same API.